### ISSUE TITLE
Introduce validations?

### ISSUE THREAD

**Issue by niku** [otoyo/pixela_ex#5]
Pixela API reference has a good validation rules.
For example, A rule `[ -~]{8,128}` is validation for tokens.

I think having validations will be good feature for pixela_ex.
But, it also has downside it will increase codes.

Would you have any idea about validations?

**Comment by otoyo** [comment]
Thank you for the good idea.
Of course adding validations will be useful for users.

Typespec covers all params so I think extending typespec by using modules is better.

**Comment by niku** [comment]
> Typespec covers all params so I think extending typespec by using modules is better.

I agree with your thoughts. However as far as I know it is designed not so strict as we want.
I will make a PR and demonstrate such cases to you.
